lib/ShortcutTrigger/index.js

```diff
@@ -41,6 +41,10 @@ class ShortcutTrigger extends Component {
   }
 
   onKeyDown(event) {
+    if (event.target.type && event.target.type === 'text') {
```

Comment:
💚
```suggestion
    if (event.target.type  === 'text') {
```
Can we just do this? If `event.target.type` is undefined, the equality check will still fail.

Reply:
Great point, that's merged!